onnxruntime/core/providers/cuda/gpu_data_transfer.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "cuda_pch.h"
+#include "core/framework/data_transfer.h"
+
+namespace onnxruntime {
+
+enum CUDAStreamType : int {
```

Comment:
```suggestion
enum class CUDAStreamType : uint8_t {
```

Reply:
This may be refactored later (it's about the execqueueid design)